packages/sources/trueusd/src/endpoint/trueusd.ts

```diff
@@ -5,14 +5,25 @@ export const NAME = 'trueusd'
 
 export const supportedEndpoints = ['trueusd']
 
+export const endpointResultPaths = {
+  trueusd: 'totalTrust',
+}
+
 export const description = 'https://core-api.real-time-attest.trustexplorer.io/trusttoken/TrueUSD'
 
 export type TInputParameters = {
-  field?: string
+  field?: string // Deprecated - kept for backwards compatability
+  chain?: string
 }
 
 export const inputParameters: InputParameters<TInputParameters> = {
+  chain: {
+    required: false,
+    description: 'Filter data to a single blockchain',
+    type: 'string',
+  },
```

Comment:
What is the default value for 'chain' if it is not specified?  Is this included in the README?

Reply:
`totalTrust` as shown in the `endpointResultPaths`
 https://github.com/smartcontractkit/external-adapters-js/pull/2033/files#diff-48766e056cd326b1f4ad54e0925c1f60e55c075b4ad3aa1a5bada3d9f17284b3R8